plannerbenchmark/postProcessing/helpers.py

```diff
@@ -40,6 +40,7 @@ def createMetricsFromNames(
         Returns a list of all metrics for which the name was specified.
     """
     metrics = []
+    goal_indices = experiment.primeGoal().indices()
     m = experiment.obstacles()[0].dim()
```

Comment:
So we still need 'm' then? If so, could we give it a more meaningful name, such as dimension_obstacle?

Reply:
Indeed, in ``helpers.py`` itself, we don't really need `m`. However, it is passed along to the parameters of the metric classes, where it is still used.
Changed the names in another commit.
Should we do the same for the `n` variable? Could be dimension_configuration.

https://github.com/maxspahn/localPlannerBench/blob/77dfe2698e27e63690b7d709fad62932eaac7eaf/plannerbenchmark/postProcessing/helpers.py#L44

Reply:
Let's keep this `n` for the moment as it is not crucial for the time being. 
As there will be an update to motionplanningscenes, we should update all names.